docs/source/merlin_commands.rst

```diff
@@ -400,6 +400,35 @@ The ``--workers`` option allows you to pass in a regular expression of names of
 
 The only currently available option for ``--task_server`` is celery, which is the default when this flag is excluded.
 
+Hosting Local Server (``merlin server``)
```

Comment:
underline this so it's a heading:

Hosting a Local Server (``merlin server``)
------------------------------------------